eng/versioning/version_client.txt

```diff
@@ -70,7 +70,7 @@ com.azure:azure-communication-chat;1.3.13;1.4.0-beta.1
 com.azure:azure-communication-common;1.2.13;2.0.0-beta.2
 com.azure:azure-communication-common-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-email;1.0.7;1.1.0-beta.1
-com.azure:azure-communication-identity;1.4.11;1.5.0-beta.1
+com.azure:azure-communication-identity;1.4.11;1.5.0
```

Comment:
should it be updated by the release PR?

Reply:
I referred previous PR that was created for CTE & custom expiration and this was updated there as well. I think we need it because we are updating version in pom and changelog file